dex/encode/encode.go

```diff
@@ -95,6 +95,14 @@ func UnixTimeMilli(msEpoch int64) time.Time {
 	return time.Unix(sec, msec*1e6).UTC()
 }
 
+// UnixTimeMilliLocal returns a Time for an elapsed time in milliseconds since
+// the Unix Epoch. The time will have Location set to the local machine time.
+func UnixTimeMilliLocal(msEpoch int64) time.Time {
+	sec := msEpoch / 1000
+	msec := msEpoch % 1000
+	return time.Unix(sec, msec*1e6).Local()
```

Comment:
I think we can do without `UnixTimeMilliLocal` and just call like `UnixTimeMilli(stamp).Local()`.  Might as well keep the `dex/encode` package API smaller since this is mostly dup code. and `time.Time`s can change locations back and forth with no trouble.